core/src/env/test_env.rs

```diff
@@ -28,6 +28,9 @@ use core::cell::{
     RefCell,
 };
 
+/// A wrapper for the generic bytearray used for data in contract events.
+struct EventData(Vec<u8>);
```

Comment:
You might want to add `#[derive(Debug, PartialEq, Eq, ...)]` here since the purpose of the test-environment is to introspect it.